File: java/test/ExistsTest.java
package voxgig.bluefintecsmerchantportalsdk.sdktest;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;

import voxgig.bluefintecsmerchantportalsdk.core.BluefinTecsMerchantPortalSDK;

public class ExistsTest {

  @Test
  public void testMode() {
    BluefinTecsMerchantPortalSDK testsdk = BluefinTecsMerchantPortalSDK.testSDK();
    assertNotNull(testsdk, "expected non-nil SDK");
  }
}

File: kotlin/test/ExistsTest.kt
package voxgig.bluefintecsmerchantportalsdk.sdktest

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test

import voxgig.bluefintecsmerchantportalsdk.core.BluefinTecsMerchantPortalSDK

class ExistsTest {

  @Test
  fun testMode() {
    val testsdk = BluefinTecsMerchantPortalSDK.testSDK()
    assertNotNull(testsdk, "expected non-nil SDK")
  }
}
